### PR TITLE
fix link and add style for tags

### PR DESCRIPTION
**Pull request by regis-leray**

### CODE REVIEW

File: services.html [hunk 1]
```diff
@@ -181,10 +188,16 @@ <h5>With the addition of business oriented language, we eliminated costly develo
                    bypassing the long and expensive development cycle.
                </p>
                <br/>
-               <p>[ big data ] [ integrations ] [ language ]</p>
+                <div class="post-tags">                              
+                    <a class="disabled">big data</a>                                                            
+                    <a class="disabled">language</a>           
+                    <a class="disabled">integrations</a>            
+                </div>
```
Comment (sergei-shabanau) [inline services.html:195]: This is what I tried before but could not figure out the style 🙁
Thank you!

File: services.html [hunk 2]
```diff
@@ -239,10 +256,17 @@ <h5>We've built a highly scalable streaming engine to automate the manual, ad ho
...
+                   <a class="disabled">streaming</a>                                          
+                   <a class="disabled">big data</a>                                          
+                   <a class="disabled">cloud</a>           
+                   <a class="disabled">integrations</a>            
+               </div>
            </div>
        </div>
 
+        <a class="anchor" id="project-4"></a>
         <div class="row portfolio-item">
```
Comment (sergei-shabanau) [inline services.html:270]: But how is this different from putting `id` on the `div` ?

Comment (regis-leray) [reply services.html:270]: i defined a specific css style
```
a.anchor {
  display: block;
  position: relative;    
  visibility: hidden;
  top: -100px;  
}

````

Comment (sergei-shabanau) [reply services.html:270]: damn, `top: -100px;`, 
now I see: invisible `<a>` and offset 🤓

it is so hacky 😲

**Pull request merged**